hello.html

```diff
@@ -0,0 +1,56 @@
+<html>
+<a href="http://hellojalenmerch.com/">Get your merchandise here! </a>
+<head>
+  <title>Welcome to Hello Jalen</title>
+  <style>
+    h1{
+      color:red;
+    }
+    p{
+      border: 10px dotted red;
+      font-size: 32px;
+    }
+  </style>
+  </head>
+  <body>
+<head>
+  <marquee>Buy now!   Contact us today! Phone: 856-678-9987 Email:hellojalenhacks@gmail.com </marquee>
+    <h1>Hello, Guests!</h1>
+      <p>
+        This website is here to help you buy Hello Jalen merchandise.
+        If you are interested,
+        <strong>follow the instructions below closely.</strong> If not,
+        <strong>LEAVE.</strong>
+        </p>
+        <style>
+        h2{
+          color:blue;
+         }
+         h2{
+          font-size: 64px;
+         }
+         p{
+        color:white;
+       font-size: 20px;
+       background: green;
+      }
+      j{
```

Comment:
While it'd be nice if we could declare arbitrary tags like this, we can't. You can, however, use "generic" tags like `<div>` along with a class, e.g.,

```html
<div class="cool-stuff">...</div>
```

Then in your stylesheet you can write

```css
div.cool-stuff {
  font-size: 30px;
  color: orange;
}
```